pom.xml

```diff
@@ -56,14 +66,19 @@
         <dependency>
             <groupId>io.jenkins.plugins</groupId>
             <artifactId>opentelemetry</artifactId>
-            <version>3.1215.vc9db_a_0b_34c2a_</version>
+            <version>${jenkins-plugin-opentelemetry.version}</version>
             <exclusions>
                 <exclusion>
+                    <!-- 2.18.0 (managed) vs 2.28.0 -->
                     <groupId>com.google.errorprone</groupId>
                     <artifactId>error_prone_annotations</artifactId>
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-jdbc</artifactId>
+        </dependency>
```

Comment:
I think this should come from the otel-api-plugin: https://github.com/jenkinsci/opentelemetry-api-plugin/blob/main/pom.xml

Its better that versions are kept in sync / not duplicated, it could possibly be moved to the database plugin instead: https://github.com/jenkinsci/database-plugin but I think otel-api is likely better